code/modules/organs/internal/heart.dm

```diff
@@ -214,4 +214,11 @@
 	. = "[pulsesound] pulse"
 
 /obj/item/organ/internal/heart/get_mechanical_assisted_descriptor()
-	return "pacemaker-assisted [name]"
\ No newline at end of file
+	return "pacemaker-assisted [name]"
+
+/obj/item/organ/internal/heart/rejuvenate(ignore_prosthetic_prefs)
+	. = ..()
+	if(!BP_IS_PROSTHETIC(src))
```

Comment:
Might be better to emulate resuscitate() and do `pulse = PULSE_NORM; handle_pulse()`

Reply:
I don't know. I'd rather let the organ be updated later. Because, if the mob is still not completely rejuvenated, it could just immediately kill the heartbeat again.